go/exorelay/exorelay_test.go

```diff
@@ -73,6 +74,7 @@ func FeatureContext(s *godog.Suite) {
 	})
 
 	s.Step(`^ExoRelay connects to Exocom$`, func() error {
+		time.Sleep(time.Millisecond * 1)
```

Comment:
K. Created an issue for it https://github.com/Originate/exosphere-sdk/issues/302